### ISSUE TITLE
Warning: The `set-output` command is deprecated and will be disabled soon

### ISSUE THREAD

**Issue by fredericrous** [ahmadnassri/action-commit-lint#229]
hi, getting this warning when running this github acction

```
The set-output command is deprecated and will be disabled soon
Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
```

 this warning will become an error within ~4 months

**Comment by ahmadnassri** [comment]
this was resolved.
